README.md

```diff
@@ -33,7 +35,7 @@ The majority of apps fall in one of the three types below. Each type uses a diff
 1. **Client-side web applications** use the JavaScript SDK with the [OAuth 2.0 Implicit Grant](https://circuit.github.io/oauth.html#implicit).
    ```javascript
    // No client_secret needed for Implicit Grant. SDK will obtain access token.
-   const client = new Circuit.Client({
+   let client = new Circuit.Client({
```

Comment:
Any reason we change it to let?

Reply:
I'm not sure, const works I will change it back. Should I change it to const on beta as well?